.rubocop.yml

```diff
@@ -1,21 +1,13 @@
+inherit_from: .rubocop_todo.yml
+
+require:
+  - rubocop-performance
+  - rubocop-rspec
+
 AllCops:
+  DisplayCopNames: true
   Exclude:
     - vendor/**/*
     - lib/slack/web/api/slack-api-ref/**/*
-
-Metrics:
```

Comment:
We basically ignore metrics everywhere, so maybe just leave this?

Reply:
Why ignore Metrics?

Reply:
Personally I think they are noise. Line length is one of those things from the 80s :) But I am also happy if we don't ignore them as long as the violations are fixed.

Reply:
As someone with a disability who is often forced to use limited screen real estate (an iPad hovering over my head as I lay on my back), I especially appreciate adherence to lines that do not exceed ~100 characters.  Also considering the ubiquity of web-based code reviews through github/gitlab where those systems impose their own visual limitations on the width available for reading the lines in a pull request, I think shorter lines are generally better (scrolling horizontally is generally a poor UX in this context).  Such limitations can be overridden with browser extensions, but I would argue that it's best to try to adhere to code that is readable in these contexts to make it inclusive and easy for everyone.  Additionally, I would argue that at least most English readers appreciate shorter lines when trying to comprehend code they have not encountered before (perhaps this is only my personal perception, however).

There are several other important Cops in the Metrics group including ClassLength and BlockLength.  I admit I usually override these, often doubling them to 200-250 depending on project.  BlockLength in rspec test is especially troublesome.  However, I would argue that any file/class/block exceeding about 350 lines becomes troublesome to navigate in modern development flows.

I wanted to raise these concerns before proceeding with ignoring Metrics altogether, but am happy to do so if that is what's preferred here.  Please advise, thanks!

Reply:
Thanks @jcraigk for the long explanation.

First, I am sorry that I am, clearly, not sensitive enough to developers with disabilities such as yourself, that's my bad, on me.

I absolutely want your valuable contributions to this and other projects, so let's make it as easy as possible moving forward and not ignore metrics. Please adjust them as you see fit and we can discuss each individual one and I'll accept anything reasonable.